config/production.json

```diff
@@ -1,6 +1,7 @@
 {
   "host": "0.0.0.0",
   "port": "PORT",
+  "JWTSecret": "",
```

Comment:
seems like it's should be "JWTSecret": "JWT_SECRET"
@velichkod ^^

Reply:
Done